src/app/Store/card/card.effects.ts

```diff
@@ -0,0 +1,49 @@
+import {Injectable} from '@angular/core';
+import {Actions, createEffect, ofType} from '@ngrx/effects';
+import {select, Store} from '@ngrx/store';
+import {of} from 'rxjs';
+import {map, switchMap, withLatestFrom} from 'rxjs/operators';
+import * as CardActions from './card.action';
+import {selectAllCards} from './card.selectors';
+
+@Injectable()
+export class CardEffects {
+
+  constructor(private readonly _actions$: Actions, private readonly _store: Store) {
```

Comment:
```suggestion
  constructor(private readonly _actions$: Actions, private readonly _store$: Store) {
```

Store est un observable

Reply:
fait